contrib/nginx.conf

```diff
@@ -0,0 +1,16 @@
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	root /var/www/html;
+
+	index index.html;
+
+	server_name _;
+
+	location / {
+		# First attempt to serve request as file, then
+		# as directory, then fall back to displaying a 404.
+		try_files $uri /index.html =404;
```

Comment:
additionally we will also need to disable caching of index.html and add some headers for proper testing:
<pre>
        expires epoch;
        add_header Cache-Control 'private, no-store, must-revalidate, max-age=0';
        add_header Strict-Transport-Security max-age=63072000;
        add_header X-Frame-Options DENY;
        add_header X-Content-Type-Options nosniff;
        add_header Content-Security-Policy "default-src 'self'; img-src 'self' data:; style-src 'self' 'unsafe-inline'; object-src 'none'";
        add_header X-XSS-Protection "1; mode=block";
</pre>

I guess the  Strict-Transport-Security should be set by ingress later on... not sure how to accomplish it though.

Reply:
Why would we want to disable caching?

Reply:
Got it, the index.html is really small and caching it would make updating the frontend harder than it should be.